<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ZSoftvolDialog</class>
 <widget class="QDialog" name="ZSoftvolDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>510</width>
    <height>199</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Softvol properties</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Control name</string>
       </property>
       <property name="buddy">
        <cstring>editName</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="editName"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Minimal dB value</string>
       </property>
       <property name="buddy">
        <cstring>spinMinDB</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QDoubleSpinBox" name="spinMinDB">
       <property name="suffix">
        <string> dB</string>
       </property>
       <property name="minimum">
        <double>-90.000000000000000</double>
       </property>
       <property name="maximum">
        <double>-0.010000000000000</double>
       </property>
       <property name="value">
        <double>-51.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Maximal dB value</string>
       </property>
       <property name="buddy">
        <cstring>spinMaxDB</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="spinMaxDB">
       <property name="suffix">
        <string> dB</string>
       </property>
       <property name="maximum">
        <double>90.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Resolution</string>
       </property>
       <property name="buddy">
        <cstring>spinResolution</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinResolution">
       <property name="minimum">
        <number>3</number>
       </property>
       <property name="maximum">
        <number>1024</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkMuteSwitch">
       <property name="text">
        <string>Mute switch</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Control channels</string>
       </property>
       <property name="buddy">
        <cstring>spinChannels</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinChannels">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>2</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>editName</tabstop>
  <tabstop>spinMinDB</tabstop>
  <tabstop>spinMaxDB</tabstop>
  <tabstop>spinResolution</tabstop>
  <tabstop>checkMuteSwitch</tabstop>
  <tabstop>spinChannels</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ZSoftvolDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>180</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ZSoftvolDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>186</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
